### PR TITLE
Init express

### PR DESCRIPTION
**Pull request by joeivans**
- Initialize the repo for Express js server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+This is a web API server application that uses the web request/response cycle to
+interface with clients and other servers to query data and provide a client with
+high-quality results.
+
+![Architecture](./docs/images/Lab07.png)
+
+Dependencies:
+- Cors
+- Dotenv
+- Express
```
Comment (ryanemmans) [inline README.md:32]: This is looking good!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Change Log
+<!-- Use this area to document the iterative changes made to your application as
+each feature is successfully implemented. Use time stamps. Here's an example:
+08-25-2021 07:44 - Initialize netlify deployment preview.
+
+## Credit and Collaborations
+<!-- Give credit (and a link) to other people or resources that helped you build
+this application. -->
+- I paired with Kason Braley for the initial design of the project.
```
Comment (SarahCreager) [inline README.md:42]: I like that you mentioned this. It's a great contribution to add to the README

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "type": "git",
+    "url": "git+https://github.com/joeivans/city-explorer-api.git"
+  },
+  "keywords": [],
+  "author": "Joe Ivans",
+  "license": "MIT",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dotenv": "^10.0.0",
+    "express": "^4.17.1"
```
Comment (ryanemmans) [inline package.json:19]: Good job on the init setup. If you'd like, you could change index.js to server.js.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+<hr />
+Name of feature: Project setup<br />
+Estimate of time needed to complete: 3h<br />
+Start time: 06:00<br />
+Finish time: 07:45<br />
+Actual time needed to complete: 1h45<br />
```
Comment (SarahCreager) [inline README.md:48]: It's always nice when you overshoot the time estimate. Great job!

**Comment by joeivans** [comment]
I pushed to the same branch as yesterday

**Review by hkimbrough22 — COMMENTED** [review]
in your data/weatherData.http I am not sure what this file is or does?

File: server.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+const cors = require('cors');
+const {response} = require('express');
+require('dotenv').config();
+const app = express();
+const PORT = process.env.PORT;
+
+app.use(cors());
+
+app.get('/health-check', (req, res) => {
+  console.log(Date.now());
```
Comment (hkimbrough22) [inline server.js:13]: Wow, this is a really cool idea. I might steal this 👍

File: server.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  res.send('healthy');
+});
+
+app.get('/photo', getPhotos);
```
Comment (hkimbrough22) [inline server.js:17]: Nice, you already implemented this from today's lesson!

**Pull request merged**